extensions/resource-deployment/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"extends": "../shared.tsconfig.json",
+	"compilerOptions": {
+		"outDir": "./out",
+		"strict": false,
+		"alwaysStrict": false,
+		"noImplicitAny": false,
+		"noImplicitReturns": false,
+		"noUnusedLocals": false,
+		"noUnusedParameters": false
```

Comment:
Why disable these options?

Reply:
good catch, I will enable and fix the errors.